### PR TITLE
Sneha, Alekhya | MOBN-949 | Avoid auto focus for status filter when w…

### PR DESCRIPTION
**Pull request by AlekhyaYalla**
…indow location is new/edit appointment

### CODE REVIEW

File: ui/react-components/components/PatientSearch/PatientSearch.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const PatientSearch = (props) => {
             placeholder={placeholder}
             selectedValue={value}
             isDisabled={isDisabled}
+            autoFocus={true}
```
Comment (bsneha90) [inline ui/react-components/components/PatientSearch/PatientSearch.jsx]: autoFocus should come from parent, we can use PatientSearch somewhere else and not have the first focus on it.

Comment (AlekhyaYalla) [reply ui/react-components/components/PatientSearch/PatientSearch.jsx]: Yes, handled the same

File: src/directives/multiSelectAutocomplete.js [hunk 2]
```diff
@@ -4,8 +4,12 @@ angular.module('bahmni.appointments')
     .directive('multiSelectAutocomplete', [function () {
         var link = function ($scope, element) {
             $scope.focusOnTheTest = function () {
-                var autoselectInput = element.find("input.input");
-                autoselectInput[0].focus();
+                const addAppointmentUrl = "#/home/manage/appointments/calendar/new";
```
Comment (binduak) [inline src/directives/multiSelectAutocomplete.js]: @AlekhyaYalla 
can we move these constants to constants file ??

Comment (AlekhyaYalla) [reply src/directives/multiSelectAutocomplete.js]: added

**Pull request merged**